CHANGELOG.md

```diff
@@ -65,6 +65,14 @@ Replace any use of this class with the `govuk-tag--grey` class.
 
 This change was introduced in [pull request #2417: Remove deprecated `govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
 
+#### Style the `html` in your cookie banner
+
+We have removed the default font styles from the cookie banner Sass, to make it more obvious when classes/styles have been forgotten.
```

Comment:
```suggestion
We've removed the default font styles from the cookie banner Sass, to make it more obvious when classes and styles have been forgotten.
```
Forgotten by what? (Sorry if daft question - let me know if the meaning would be obvious to users!)

Reply:
This is talking about classes/styling being forgotten on the HTML that's being passed into the cookie banner. Some users may assume that the HTML they pass in is somehow automatically styled. This isn't the case so we want to make it really obvious if users have forgotten to add their own styles.